site/content/_index.md

```diff
@@ -80,16 +80,16 @@ apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 
 metadata:
-  name: basic-cluster
-  region: eu-north-1
+    name: basic-cluster
+    region: eu-north-1
 
 nodeGroups:
-  - name: ng-1
-    instanceType: m5.large
-    desiredCapacity: 10
-  - name: ng-2
-    instanceType: m5.xlarge
-    desiredCapacity: 2
+    - name: ng-1
+      instanceType: m5.large
+      desiredCapacity: 10
+    - name: ng-2
+      instanceType: m5.xlarge
+      desiredCapacity: 2
```

Comment:
Can you change the yaml indentation back to 2 spaces?

Reply:
Yes, no problem